vispy/visuals/visual.py

```diff
@@ -30,6 +30,11 @@ class Visual(object):
     bounds_change : Event
         Emitted when the bounds of the visual have changed.
 
+    Notes
```

Comment:
The sentence doesn't say it can only be used within the scenegraph, (on the contrary), it states that in order to be used within it, it gets mixed with `vispy.scene.Node`.